src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,177 @@
-import React from 'react';
+/* eslint-disable no-useless-return */
+import classNames from 'classnames';
+import React, {
+  FormEvent,
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+import { InputErrors } from '../../enums/InputErrors';
+import { findFilmByTitle } from '../../api/api';
+import { MovieCard } from '../MovieCard';
+
 import './FindMovie.scss';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+type Props = {
+  setMovies: React.Dispatch<React.SetStateAction<Movie[]>>;
+};
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie: React.FC<Props> = ({
+  setMovies,
+}) => {
+  const [error, setError] = useState(InputErrors.None);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [titleTextPattern, setTitleTextPattern] = useState('');
+  const [findedMovie, setFindedMovie] = useState<Movie | null>(null);
+
+  const handleSubmit = useCallback((e: FormEvent) => {
+    e.preventDefault();
+
+    if (!findedMovie) {
+      setError(InputErrors.CantAddFirstlyFindSomeFilm);
+
+      return;
+    }
+
+    setMovies(prevValue => {
+      if (prevValue.find(
+        film => film.imdbID === findedMovie.imdbID,
+      ) !== undefined) {
```

Comment:
Object converts to true 

```suggestion
      if (prevValue.find(
        film => film.imdbID === findedMovie.imdbID,
      )) {
```